src/library/scala/sys/process/ProcessBuilder.scala

```diff
@@ -236,6 +236,7 @@ trait ProcessBuilder extends Source with Sink {
    * process exits with a non-zero value, the LazyList will provide all lines up
    * to termination but will not throw an exception.
    */
+  @annotation.nowarn("cat=lint-multiarg-infix")  // should the warning trigger just because the method name ends in punctuation?! currently it does
```

Comment:
I was thinking that `crazyLines_:` is a suspicious name but actually multiargs is of course only for left-assoc:

`log crazyLines_: builder`

Maybe nothing about `id_op` suggests infix.

Reply:
BTW the other hope was that a warning would hasten its exit to a module.